papaya-core/src/main/java/pl/edu/agh/papaya/service/project/ProjectService.java

```diff
@@ -57,6 +58,14 @@ public boolean isUserInProject(Project project, String userId) {
                 .anyMatch(userId::equals);
     }
 
+    public List<Long> getAllUsersIdInProject(Project project) {
```

Comment:
`getAllUserIdsInProject`?

Reply:
Plus nie wiem, czy nie przemawiałaby do mnie bardziej metoda, która tutaj Ci wyciąga userów z projektu, a nie już konkretnie ich id

Reply:
I na pewno powinni gdzieś być filtrowani userzy z rolą `INACTIVE`

Reply:
to `getAllUsersInProject`, `getAllActiveUsersInProject` bym proponowal